README.md

```diff
@@ -676,6 +675,7 @@ A curated list of awesome Java frameworks, libraries and software.
 
 *Libraries for developing reactive applications.*
 
+* [Akka](http://akka.io) - Toolkit and runtime for building concurrent, distributed, and fault tolerant event-driven applications.
```

Comment:
The description should probably say something like, "Toolkit and runtime for building concurrent, reactive, distributed, and fault tolerant event-driven applications."